src/main/java/com/troubleshooting/model/IAuthnUserDataModel.java

```diff
@@ -0,0 +1,13 @@
+package com.troubleshooting.model;
+
+public interface IAuthnUserDataModel {
+
+    String getCtnId();
```

Comment:
What is Ctn? if the method has no documentation the the name should clearly say what is does, and if it's public it's better to have both.

Reply:
ctn = container id- updated to `getContainerId`